### ISSUE TITLE
Errors when using jasmine-marbles with Jest

### ISSUE THREAD

**Issue by mcblum** [synapse-wireless-labs/jasmine-marbles#53]
Hey! So I really prefer `jasmine-marbles` to `jest-marbles` but in my current project I am seeing an issue:

```
node_modules/@types/jest/index.d.ts:730:15 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

730     interface Matchers<R, T> {
                  ~~~~~~~~

node_modules/jasmine-marbles/index.d.ts:26:15 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

26     interface Matchers<R> {
                 ~~~~~~~~

Found 2 errors.
```
Do you happen to know if there's an easy workaround for this?

**Comment by PhilippMeissner** [comment]
Use jest-marbles.

**Comment by brandonroberts** [comment]
Should be resolved in `0.7.0`
